cmd/neofs-cli/modules/container/attributes.go

```diff
@@ -0,0 +1,223 @@
+package container
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	internalclient "github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/client"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/common"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/commonflags"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/key"
+	"github.com/nspcc-dev/neofs-sdk-go/client"
+	apistatus "github.com/nspcc-dev/neofs-sdk-go/client/status"
+	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	neofscrypto "github.com/nspcc-dev/neofs-sdk-go/crypto"
+	neofsecdsa "github.com/nspcc-dev/neofs-sdk-go/crypto/ecdsa"
+	"github.com/spf13/cobra"
+)
+
+// Set attribute command flags.
+const (
+	setAttributeNameFlag     = "attribute"
+	setAttributeValueFlag    = "value"
+	setAttributeValidForFlag = "valid-for"
+)
+
+var setAttributeFlagVars struct {
+	id        string
+	attribute string
+	value     string
+	validFor  time.Duration
+}
+
+var setAttributeCmd = &cobra.Command{
+	Use:   "set-attribute",
+	Short: "Set attribute for container",
+	Long:  "Set attribute for container",
+	Args:  cobra.NoArgs,
+	RunE:  setAttribute,
+}
+
+func initSetAttributeCmd() {
+	commonflags.Init(setAttributeCmd)
+
+	flags := setAttributeCmd.Flags()
+	flags.StringVar(&setAttributeFlagVars.id, commonflags.CIDFlag, "", commonflags.CIDFlagUsage)
+	flags.StringVar(&setAttributeFlagVars.attribute, setAttributeNameFlag, "", "attribute to be set")
+	flags.StringVar(&setAttributeFlagVars.value, setAttributeValueFlag, "", "value for the attribute")
+	flags.DurationVar(&setAttributeFlagVars.validFor, setAttributeValidForFlag, 0, "request validity duration")
+
+	for _, f := range []string{
+		commonflags.CIDFlag,
+		setAttributeNameFlag,
+		setAttributeValueFlag,
+		setAttributeValidForFlag,
```

Comment:
Shouldn't be required, have some sane default for it. Users don't care much about these things.